playwright/src/main/java/com/microsoft/playwright/junit/DefaultPlaywrightFactory.java

```diff
@@ -0,0 +1,10 @@
+package com.microsoft.playwright.junit;
+
+import com.microsoft.playwright.Playwright;
+
+class DefaultPlaywrightFactory implements PlaywrightFactory {
```

Comment:
They don't have to be public since they are only used internally.  but we can make them public if you'd like

Reply:
> They don't have to be public since they are only used internally. 

Can people use use the explicitly in the annotations today (`@UsePlaywright(playwright = DefaultPlaywrightFactory.class)`) ?

> but we can make them public if you'd like

I see 2 reasons to make them public: people may want to subclass them, they may want to use them explicitly in the annotations.

Reply:
Sure I can make them public. I figured no one would explicitly use the default ones and would define their own if they wanted something custom.